docker-compose.test.yml

```diff
@@ -1,34 +1,18 @@
 ---
 services:
-  redis:
```

Comment:
So it turns out that you can remove duplication from docker-compose files by chaining them together

https://docs.docker.com/compose/extends/#understanding-multiple-compose-files

Which means that we don't have to maintain duplicate configurations for all the containers that stay the same between the two tests. We just overwrite pgsql and api.